activestorage/app/assets/javascripts/activestorage.js

```diff
@@ -655,6 +655,7 @@
         if (status >= 200 && status < 300) {
           this.callback(null, response);
         } else {
+          console.error(`Error Storing ${this.file.name}, Error Message: ${this.xhr.response}`)
```

Comment:
You're editing the compiled `activestorage.js` file directly here. Changes need to go through the source files first: https://github.com/rails/rails/blob/96e69b5a33f44e5339345ef3c4871aba51a1e938/activestorage/app/javascript/activestorage/blob_upload.js#L23-L34